plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/SdkUiMessages.java

```diff
@@ -4,6 +4,9 @@
 import org.eclipse.osgi.util.NLS;
 
 public class SdkUiMessages extends NLS {
+  
+  // todo Ick! Mutable public fields. Use the other kind of messages.
```

Comment:
Yes, and they can be set from other places too. These are all public and mutable.